picoquic/util.h

```diff
@@ -69,6 +72,19 @@ int picoquic_get_input_path(char * target_file_path, size_t file_path_max, const
 #define MAX(a, b) ((a) > (b) ? (a) : (b))
 #endif
 
+/* Return a directory path based on solution dir and file name */
+#ifdef _WINDOWS
```

Comment:
Wrong comment. That should be a comment for the function, not for the separator definitions.